### PR TITLE
SUS-1908 | do not filter out blocked wikis

### PR DESCRIPTION
**Pull request by idradm**

### CODE REVIEW

File: includes/wikia/services/WikiService.class.php [hunk 1]
```diff
@@ -881,8 +881,7 @@ public function getDetails( Array $wikiIds = null ) {
 				),
 				array(
 					'city_list.city_public' => 1,
-					'city_list.city_id' => $wikiIds,
-					'((city_visualization.city_flags & ' . self::FLAG_BLOCKED . ') != ' . self::FLAG_BLOCKED . ' OR city_visualization.city_flags IS NULL)'
```
Comment (mszabo-wikia) [inline includes/wikia/services/WikiService.class.php:885]: Let's get rid of the now unused constant as well.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiService.class.php#L3](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L3): Class "WikiService" has 28 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiService.class.php#L26](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L26): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [WikiService.class.php#L64](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L64): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiService.class.php#L442](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L442): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiService.class.php#L838](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L838): Remove this unused "$index" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MINOR][MINOR] [WikiService.class.php#L99](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L99): Immediately return this expression instead of assigning it to the temporary variable "$userIds". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [WikiService.class.php#L571](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L571): Immediately return this expression instead of assigning it to the temporary variable "$editsCount". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![INFO][INFO] [WikiService.class.php#L23](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L23): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [WikiService.class.php#L26](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L26): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [WikiService.class.php#L299](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L299): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO'
[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

File: includes/wikia/services/WikiService.class.php (hunk 1 above)
Comment (ghost) [reply includes/wikia/services/WikiService.class.php:885]: Removed.

**Review by ghost — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [WikiService.class.php#L3](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L3): Class "WikiService" has 28 methods, which is greater than 20 authorized. Split it into smaller classes. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)
1. ![MAJOR][MAJOR] [WikiService.class.php#L26](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L26): Remove this commented out code. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)
1. ![MAJOR][MAJOR] [WikiService.class.php#L64](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L64): Reduce the number of returns of this function 4, down to the maximum allowed 3. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)
1. ![MAJOR][MAJOR] [WikiService.class.php#L442](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L442): Replace or remove this deprecated Wikia::log logging statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)
1. ![MAJOR][MAJOR] [WikiService.class.php#L838](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L838): Remove this unused "$index" local variable. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)
1. ![MINOR][MINOR] [WikiService.class.php#L99](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L99): Immediately return this expression instead of assigning it to the temporary variable "$userIds". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [WikiService.class.php#L571](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L571): Immediately return this expression instead of assigning it to the temporary variable "$editsCount". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![INFO][INFO] [WikiService.class.php#L23](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L23): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [WikiService.class.php#L26](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L26): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)
1. ![INFO][INFO] [WikiService.class.php#L299](https://github.com/Wikia/app/blob/a9b07c01766e336ab65c119ddd18f2d751c22482/includes/wikia/services/WikiService.class.php#L299): Move this trailing comment on the previous empty line. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**